Tasks/DownloadBuildArtifactsV0/package-lock.json

```diff
@@ -9,9 +9,9 @@
       "resolved": "https://registry.npmjs.org/align-text/-/align-text-0.1.4.tgz",
       "integrity": "sha1-DNkKVhCT810KmSVsIrcGlDP60Rc=",
       "requires": {
-        "kind-of": "3.2.2",
-        "longest": "1.0.1",
-        "repeat-string": "1.6.1"
+        "kind-of": "^3.0.2",
```

Comment:
Undo changes in this file .. these are unrelated to DownloadGitHubRelease task

Reply:
removed